lib/lotus/model/adapters/sql/query.rb

```diff
@@ -663,6 +660,11 @@ def apply(query)
             end
           end
 
+          def _push_to_conditions(condition_type, condition)
+            raise ArgumentError, 'You need to specify a condition.' unless condition
```

Comment:
I agree with @AlfonsoUceda since `nil || nil == nil`. If we remove the nil check for the second argument we may end with a nil value being propagated to `conditions`